modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/Fields.tsx

```diff
@@ -281,6 +281,15 @@ const SaveFDSFieldsModalContent = ({
 							<ManagementToolbar.ItemList expand>
 								<ManagementToolbar.Item className="pr-2">
 									<ClayCheckbox
+										aria-label={
+											!isSelectAllChecked()
+												? Liferay.Language.get(
+														'select-all-fields'
+												  )
+												: Liferay.Language.get(
+														'unselect-fields'
+												  )
+										}
```

Comment:
Neither of these language keys exist in DXP. If you look in DOM, you'll see `aria-label` resolves to `select-all-fields`. How screen reader reads this, I have no idea :slightly_smiling_face:  

You can do one of two things:
1. Use existing language keys `select-all` and `deselect-all`.
2. Add two new keys `select-all-fields` and `unselect-all-fields` to [Language.properties](https://github.com/liferay/liferay-portal/blob/master/modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties)